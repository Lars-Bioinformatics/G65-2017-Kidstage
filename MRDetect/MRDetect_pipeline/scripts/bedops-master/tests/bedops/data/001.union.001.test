chr1	1	10
chr1	2	8
chr10	500	600
chr11	4	600
chr12	3	4
chr12	101	178
chr13	1	2
chr14	59	78
chr15	500	699
chr16	4	5
chr17	5	77
