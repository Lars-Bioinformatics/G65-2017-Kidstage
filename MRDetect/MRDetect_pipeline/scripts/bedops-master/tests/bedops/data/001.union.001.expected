chr1	0	108
chr1	0	110
chr10	400	700
chr11	0	700
chr12	0	104
chr12	1	278
chr13	0	102
chr14	0	178
chr15	400	799
chr16	0	105
chr17	0	177
